olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/PrefetchTilesRequest.h

```diff
@@ -149,6 +149,35 @@ class DATASERVICE_READ_API PrefetchTilesRequest final {
     return *this;
   }
 
+  /**
+   * @brief Changes the prefetch behavior when prefetching a list of tiles. In
+   * case the tile does not exists, prefetch algorithm searches for the nearest
+   * parent and prefetch it.
```

Comment:
```suggestion
   * @brief Changes the prefetch behavior when prefetching a list of tiles.
   *
   * In case a tile does not exist, the prefetch algorithm searches for the nearest
   * parent and prefetches it.
```